fuzz/main.go

```diff
@@ -56,9 +55,10 @@ func main() {
 	go func() {
 		for range c {
 			p := message.NewPrinter(language.English)
+			p.Println()
 			p.Printf("WM operations:     %d\n", wmCount)
```

Comment:
It could just be `p.Printf("\nWM operations:     %d\n", wmCount)`, but this seems good as well

Reply:
I'll leave it for now. I hope to add back vterm stats later. Right now it's always zero, so I thought I might as well remove it from the output